### PR TITLE
feat(Automation) Move qentelli automation testbed

### PR DESCRIPTION
**Pull request by NidhiDixit09**
## Reference
SDK-1972 -- Move RN test app from qentelli repo to RN repo
https://branch.atlassian.net/browse/SDK-1972 

## Summary
* Moved React Native test bed app from https://github.com/BranchMetrics/qentelli-saas-sdk-testing-react-native-testbed to here. This will make GHA execution easier.
* Added workflow for running Appium tests

## Type Of Change

- [ ] New feature (non-breaking change which adds functionality)

## Testing Instructions

- Review workflow
- Run Appium-automation workflow and verify if all tests are passed.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

File: .github/workflows/automation-trigger-test.yml
Comment (gdeluna-branch) [inline .github/workflows/automation-trigger-test.yml:29]: I approved the automation side, once this is pointed to master i will approve this.

Comment (JagadeeshKaricherla-branch) [reply .github/workflows/automation-trigger-test.yml:29]: GHAs look good to me

File: branch-sdk-automation-testbed/.vscode/settings.json
Comment (JagadeeshKaricherla-branch) [inline branch-sdk-automation-testbed/.vscode/settings.json:1]: do we need this vscode settings file inside the repo ?

Comment (NidhiDixit09) [reply branch-sdk-automation-testbed/.vscode/settings.json:1]: @JagadeeshKaricherla-branch I am not sure. I just copied code from qentelli repo as it is. Whats your suggestion ? Should we remove it ?

Comment (JagadeeshKaricherla-branch) [reply branch-sdk-automation-testbed/.vscode/settings.json:1]: These can be added to gitignore file later

**Review by JagadeeshKaricherla-branch — APPROVED** [review]

**Pull request merged**
